cloudify_helm/configuration.py

```diff
@@ -104,7 +104,8 @@ class FileContentConfiguration(KubernetesConfiguration):
     def _get_kubeconfig(self):
         if self.FILE_CONTENT_KEY in self.configuration_data:
             file_content = self.configuration_data[self.FILE_CONTENT_KEY]
-            with tempfile.NamedTemporaryFile(delete=False) as tmp_file:
+            with tempfile.NamedTemporaryFile(delete=False,
+                                             mode='w') as tmp_file:
```

Comment:
Are we 100% ok with this being deleted after the context closes?

Reply:
Its not being deleted because of `delete=False`